### PR TITLE
Add new "Meanings Of Life" to the list.

### PR DESCRIPTION
**Pull request by rozzzly**
OP put "cocaine" in there. I nominate another—perhaps more relevant to the field of computer science—drug for the list of "meanings of life."

---

While I'm at it... 
## 42

[**Scene From the Movie**](https://www.youtube.com/watch?v=aboZctrHfK8)

And whilst finding the quote, I came across [one redditor's theory](https://www.reddit.com/r/FanTheories/comments/24uio5/the_truth_behind_42_being_the_meaning_of_life_the/). Spooky coincidence or secret plot of the American Standard Code for Information Interchange (ASCII)? I'll let you decide :wink:

**[Quotation](https://www.goodreads.com/quotes/tag/42) from the book**

> "the task we have designed you to perform is this. We want you to tell us...." he paused, "The Answer."
> "The Answer?" said Deep Thought. "The Answer to what?"
> "Life!" urged Fook.
> "The Universe!" said Lunkwill.
> "Everything!" they said in chorus.
> Deep Thought paused for a moment's reflection.
> "Tricky," he said finally.
> "But can you do it?"
> Again, a significant pause.
> "Yes," said Deep Thought, "I can do it."
> "There is an answer?" said Fook with breathless excitement.
> "Yes," said Deep Thought. "Life, the Universe, and Everything. There is an answer. But, I'll have to think about it."
> [...]
> “How long?” he said.
> “Seven and a half million years,” said Deep Thought.
> [...] 
> "Seven and a half million years...!” they cried in chorus.
> “Yes,” declaimed Deep Thought, “I said I’d have to think about it, didn’t I?"
> [...]
> [Seven and a half million years later.... Fook and Lunkwill are long gone, but their ancestors continue what they started] _[sic]_
> "We are the ones who will hear," said Phouchg, "the answer to the great question of Life....!"
> "The Universe...!" said Loonquawl.
> "And Everything...!"
> [...]
> Loonquawl nervously [said], "do you have...er, that is..."
> "An Answer for you?" interrupted Deep Thought majestically. "Yes, I have."
> The two men shivered with expectancy. Their waiting had not been in vain.
> "There really is one?" breathed Phouchg.
> "There really is one," confirmed Deep Thought.
> "To Everything? To the great Question of Life, the Universe and everything?"
> "Yes."
> [...]
> "You're really not going to like it," observed Deep Thought.
> "Tell us!"
> "All right," said Deep Thought. "The Answer to the Great Question..."
> "Yes..!"
> "Of Life, the Universe and Everything..." said Deep Thought.
> "Yes...!"
> "Is..." said Deep Thought, and paused.
> "Yes...!"
> "Is..."
> "Yes...!!!...?"
> "Forty-two," said Deep Thought, with infinite majesty and calm.”

### CODE REVIEW

File: meanings.h [hunk 1]
```diff
@@ -24,6 +24,8 @@ static Meaning meanings[] = {
     {"management", "Management", "Delegate."},
     {"fengshui", "Feng Shui", "Harmonize."},
     {"coke", "Cocaine Addict", "Just one more line!"},
```
Comment (rozzzly) [inline meanings.h:26]: The line of OP's  code I mentioned in the PR.

**Comment by jkingsman** [comment]
Thanks for the PR. I can see where you're coming from with 42; I'd agree with that - your PR does seem to have a typo, though: I think it should say 'To answer [...]'. 42, however, could be seen as the meaning itself, depending on your interpretation. Perhaps have finding the answer be the meaning of the philosophy 42, and vice versa?

In regards to Adderall, I'm not sure that a pure definition fits with the spirit of one liners that could be answers to the questions "Why am I here?" and "What is the meaning of life?" If you can figure out a way to rephrase it to fit that better, I'd be happy to include it. As a nitpicky note, Adderall is actually a mixture of dextro and levo; Dexedrine would be pure dextro.
